page.go

```diff
@@ -403,7 +403,23 @@ type gstate struct {
 
 // Content returns the page's content.
 func (p Page) Content() Content {
-	strm := p.V.Key("Contents")
+	switch v := p.V.Key("Contents"); v.Kind() {
+	case Stream:
+		return p.contentForStream(v)
+	case Array:
+		var c Content
+		for i := 0; i < v.Len(); i++ {
+			cfs := p.contentForStream(v.Index(i))
+			c.Text = append(c.Text, cfs.Text...)
+			c.Rect = append(c.Rect, cfs.Rect...)
+		}
+		return c
+	default:
+		panic("bad content kind")
```

Comment:
I think this panic here might cause undocumented and unexpected problems for users. I was using the package earlier on the bus back from school and wrote code that invoked p.Content() without explicitly checking the Kind(). Perhaps we can just make an explicit check for Array then let the contentForStream handle be the catch all?